citation_template.html.rsp

```diff
@@ -1,59 +1,102 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
-	<!-- Required meta tags -->
+    <!-- Required meta tags -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    
+
     <!-- Bootstrap CSS -->
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-F3w7mX95PdgyTmZZMECAngseQB83DfGTowi0iMjiWaeVhAn4FJkqJByhZMI3AhiU" crossorigin="anonymous">
 
+    <style>
+        body {
+            background-color: #20c9c9;
+            background-image: url("bg-photo.jpg");
+            background-position: center center;
+            background-repeat: no-repeat;
+            background-attachment: fixed;
+            background-size: cover;
+        }
+        main {
+            word-break: break-word;
+            width: calc(100% - 2rem);
+            background-color: white;
+        }
+        .site-header {
+            background-color: #4d4d4d;
+            color: white;
+        }
+        .all-caps-link {
+            display: inline-block;
+            text-transform: uppercase;
+            text-decoration-line: none;
+            color: unset;
+        }
+        .all-caps-link:hover {
+            color: unset;
+        }
+        .attachment + .attachment {
+            margin-top: 3rem;
+        }
+    </style>
```

Comment:
I'd strongly recommend extracting these custom styles to their own CSS file, but I didn't take that approach (entirely out of laziness)